### ISSUE TITLE
[Components and pattern standards] The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA. (04.01.1)

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/va.gov-team#51009]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Authenticated 22-1990e Transfer of Entitlement (TOE) Form Updates

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA.

### Issue Details
Breadcrumb for this form does not reflect the accurate hierarchy and position of this page - its missing the "survivor and dependent benefits" segment.

### VA.gov Experience Standard
[Category Number 04, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4 A

### Platform Recommendation
This breadcrumb needs to be updated by the education team prior to launch. This breadcrumb issue is launch blocking and will trigger another launch-blocking issue: that there is no entry point for this form on the "VA education benefits for survivors and dependents" page. Please resolve the breadcrumb issue and review the other additional ticket.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by sterkenburgsara** [comment]
FYSA @shiragoodman @mnorthuis

**Comment by btruongafs** [comment]
@sterkenburgsara @shiragoodman wasn't sure for launch blocking issue fixes if yall need to review them before closure, but this issue has been fixed in staging.

**Comment by shiragoodman** [comment]
thanks @btruongafs . We don't require that you validate with us. We use the honor system. If you say it's fixed and your Product Owner agrees, then that's good with us. However, if you need _help_ validating it's fixed or if you'd _like us_ to validate, that's something you can ask of us.

**Comment by btruongafs** [comment]
@shiragoodman thanks for the clarification. we're good on our end, so closing out this ticket.
